src/controllers/userProfileController.js

```diff
@@ -233,9 +233,9 @@ const userProfileController = function (UserProfile) {
 
   const putUserProfile = async function (req, res) {
     const userid = req.params.userId;
-    const isRequestorAuthorized = !!(
+    const isRequestorAuthorized = (!!(
       canRequestorUpdateUser(req.body.requestor.requestorId, userid) && (
-        await hasPermission(req.body.requestor, 'putUserProfile')
+        await hasPermission(req.body.requestor, 'putUserProfile')) || await hasPermission(req.body.requestor, 'requestBio')
```

Comment:
This makes the `requestBio` permission also able to modify the rest of users' profiles, not just the `bioRequested` field.

Reply:
Changed.